inc/sccm.class.php

```diff
@@ -635,7 +637,24 @@ static function executePush($task) {
                         Toolbox::logInFile('sccm', "ERROR RETURNED : ".$body." \n", true);
                      } else {
                         $task->addVolume(1);
+
+                        if ($PluginSccmConfig->getField('use_lasthwscan') == 1) {
+                           $plugin = new Plugin();
+                           if ($plugin->isActivated("fusioninventory")) {
+                              $data = PluginFusioninventoryAgent::getByDeviceID($tab['CSD-MachineID']);
+                              $pfInventoryComputerComputer = new PluginFusioninventoryInventoryComputerComputer();
+                              $a_computerextend = $pfInventoryComputerComputer->hasAutomaticInventory($data['computers_id']);
+                              if (count($a_computerextend) > 0){
+                                 $a_computerextend['last_fusioninventory_update'] = $tab['vWD-LastScan']->format('Y-m-d h:i');
+                                 $pfInventoryComputerComputer->update($a_computerextend);
+                              }
+                           } else {
+                              Toolbox::logInFile('sccm', "Push WARNING - Can't update fusioninventory last inventory, plugin is disabled"."\n", true);
```

Comment:
Maybe this log should be only done once per push. If you have many machines to push, your log file will be polluted by these logs.

Reply:
move plugin check on top